### DISCUSSION TITLE
warning in IDE when code style is incorrect

### DISCUSSION THREAD

**Discussion by Lirrrx** [biomejs/biome#2612] [Q&A]
For example, I have formatter.quoteStyle: "singe" and I want to see the warning in IDE (vscode) when i wrote double quote. Is it possible right now?

**Comment by ematipico** [comment]
It isn't possible, and we don't plan to provide this sophisticated feature.

Usually, in the IDE you configure a formatter to run when you save or type, and it takes care of it. If you want to add some checks, you should use the CLI
